dequant.py

```diff
@@ -4,9 +4,9 @@
 from tqdm import tqdm
 
 def dequantize_tensor(tensor, dtype=None, dequant_dtype=None):
-    data = torch.tensor(tensor.data)
+    data = tensor.data.clone().detach()

```

Comment:
I was doing this on purpose so the extra attributes don't have to be copied for all the ops inside the dequant kernels. https://github.com/city96/ComfyUI-GGUF/issues/66
 .clone() is disabled and .detach() also doesn't do anything since we don't have gradients so you could just put "tensor.data" and it'd be the same lol.